src/lib/background/state/sagas/index.js

```diff
@@ -1,20 +1,24 @@
 "use strict";
 
-const {fork} = require("redux-saga/effects");
 const {watchForFailuresAndLogError} = require("./failureLogger");
 const {watchForSubmitFlattrs} = require("./submitFlattrs");
 const {watchForNewVisits} = require("./saveVisitTimestamps");
-const {watchForDomainUpdates} = require("./updateDomainsList");
+const {
+  watchForDomainUpdateAlarm,
+  watchForDomainUpdateStart
+} = require("./updateDomainsList");
 
-let sagas = [
-  watchForFailuresAndLogError.bind(null, {}),
-  watchForDomainUpdates,
-  watchForNewVisits,
-  watchForSubmitFlattrs
-];
+let sagas = [];
 
-function* mainSaga()
+if (typeof global.chrome !== "undefined")
```

Comment:
> However, in this particular case it may be even better not to include this module in the tests in the first place if we don't intend to run them.

I'll have to do that for every pre existing test though, and we'll have to remember to do this for every new test, so I think that would be too much to deal with.

Reply:
I see what you mean… that'd be quite troublesome indeed. Nevermind then.